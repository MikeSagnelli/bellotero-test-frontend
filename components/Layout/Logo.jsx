import React from 'react';
import { LogoContainer, MainLogo } from './styles';

const Logo = () => (
  <LogoContainer>
    <MainLogo src="/static/images/bellotero.svg" alt="Bellotero" />
  </LogoContainer>
);

export default Logo;
