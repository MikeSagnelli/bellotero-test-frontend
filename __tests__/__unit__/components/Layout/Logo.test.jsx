import React from 'react';
import { shallow } from 'enzyme';
import Logo from '../../../../components/Layout/Logo';

describe('Test Logo', () => {
  it('renders logo', () => {
    const component = shallow(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
