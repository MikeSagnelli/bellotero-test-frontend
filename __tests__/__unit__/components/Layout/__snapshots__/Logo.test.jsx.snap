// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Logo renders logo 1`] = `
<styles__LogoContainer>
  <styles__MainLogo
    alt="Bellotero"
    src="/static/images/bellotero.svg"
  />
</styles__LogoContainer>
`;
